products/compute/terraform.yaml

```diff
@@ -20,12 +20,18 @@ overrides: !ruby/object:Provider::ResourceOverrides
       name: "address_basic"
       vars_documentation:
         address_name: "my-address"
+      vars_test:
```

Comment:
Wondering if instead of having vars_test and vars_documentation, it might be better to define just one hash for example attributes and an expected suffix (i.e. "address_name" : "address") and then having doc_name and test_name methods that generate "%s-{:prefix}" and "my-{:prefix}" as needed.